### PR TITLE
Recommended fixes performed. Added front-end components and webpack.

### PR DESCRIPTION
**Pull request by fyun89**

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE <!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <title>Run the Rentway</title>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```
Comment (etcaples) [inline client/dist/index.html]: Why are you including these sources in your main html? This seems necessary to me.

File: client/dist/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE <!DOCTYPE html>
```
Comment (etcaples) [inline client/dist/index.html]: This line's syntax is wonky.

File: client/src/components/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render () {
+    const carouselMain = {
```
Comment (etcaples) [inline client/src/components/Carousel.jsx]: Is it necessary to put the CSS in your react component like this? Are you able to put it all in your stylesheet?

File: client/src/components/CarouselListEntries.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (etcaples) [inline client/src/components/CarouselListEntries.jsx]: I believe that importing reactDOM at the beginning of every component is necessary unless you're actually calling it.

File: client/dist/index.html (hunk 1 above)
Comment (etcaples) [reply client/dist/index.html]: *seems unneccessary

File: client/src/components/CarouselListEntries.jsx (hunk 4 above)
Comment (etcaples) [reply client/src/components/CarouselListEntries.jsx]: *is unnecessary

File: client/dist/index.html (hunk 2 above)
Comment (fyun89) [reply client/dist/index.html]: Oh.... lol thanks for catching this

File: client/dist/index.html (hunk 1 above)
Comment (fyun89) [reply client/dist/index.html]: Interesting.. For some reason I was having trouble loading React components without this. But after testing out without it just now, it seems to work :/. Thanks!

File: client/src/components/Carousel.jsx (hunk 3 above)
Comment (fyun89) [reply client/src/components/Carousel.jsx]: Yes, I could put these in a CSS file. But I learned that you could do styling this way just today and I liked how I can directly see how I'm styling the relevant component in one screen this way. I'll probably move these into one CSS file when I'm cleaning up my code. Thanks for the suggestion!

File: client/src/components/CarouselListEntries.jsx (hunk 4 above)
Comment (fyun89) [reply client/src/components/CarouselListEntries.jsx]: You're right. I probably shouldn't put that. Thank you for the suggestion.

File: client/src/components/Carousel.jsx (hunk 3 above)
Comment (etcaples) [reply client/src/components/Carousel.jsx]: That makes sense.

**Pull request merged**